src/Sylius/Behat/spec/Context/Setup/OrderContextSpec.php

```diff
@@ -157,6 +160,33 @@ function it_adds_single_item_by_customer(
 
         $objectManager->flush()->shouldBeCalled();
 
-        $this->theCustomerBoughtSingle($product);
+        $this->theCustomerBoughtSingleProduct($product);
+    }
+
+    function it_adds_single_product_variant_as_item_by_customer(
+        FactoryInterface $orderItemFactory,
+        OrderInterface $order,
+        OrderItemInterface $item,
+        OrderItemQuantityModifierInterface $itemQuantityModifier,
+        SharedStorageInterface $sharedStorage,
+        ProductVariantInterface $variant,
+        ObjectManager $objectManager
+    ) {
+        $sharedStorage->get('order')->willReturn($order);
+
+        $orderItemFactory->createNew()->willReturn($item);
+
+        $variant->getPrice()->willReturn(1234);
+
+        $itemQuantityModifier->modify($item, 1)->shouldBeCalled();
+
+        $item->setVariant($variant)->shouldBeCalled();
+        $item->setUnitPrice(1234)->shouldBeCalled();
+
+        $order->addItem($item)->shouldBeCalled();
+
+        $objectManager->flush()->shouldBeCalled();
```

Comment:
IMO this it lengthy and should be squashed. I would group it by `willReturn` and `shouldBeCalled`.